articles/introduction-to-reinforcement-learning/index.md

```diff
@@ -0,0 +1,179 @@
+
+
+#### Robotics
+RL robots control Google’s data center to manage part of its [cooling infrastructure.](/https://deepmind.com/blog/article/safety-first-ai-autonomous-data-centre-cooling-and-industrial-control/)
+
+The AI robots exploits a similar approach to that used in AlphaGo, AlphaGo Zero and MuZero. This is by learning through trial and error.
+
+The algorithm worked by:
+1. Feeding it information gathered from the cooling data centers.
+2. Leaving it to determine which cooling configurations would reduce energy consumption.
+
+Google’s data center energy spending reduced by up to 40%.
+
+#### Trading and Finance
+RL agents are used to predict stock prices as well as predicting future sales. They can decide on whether to buy, hold or sell a particular stock.
+
+A great example would be the IBM’s financial trading platform which uses a RL agent for trading. It computes a reward based on the profit or losses made in every financial transaction.
+
+
+### Final Thoughts
+Reinforcement Learning is one of the most important subfield of Artificial Intelligence. Hopefully, this article has made you curious to dive deep into Reinforcement Learning.
+
+If Machine Learning is still new to you, feel free to [download and read this e-book on Machine Learning](/https://books.google.co.ke/books?hl=en&lr=&id=u8OWDwAAQBAJ&oi=fnd&pg=PR6&dq=introduction+to+machine+learning&ots=huhHR1ZCOT&sig=9hlNXYWvRqWRJgY_6P_QurJn3ac&redir_esc=y#v=onepage&q=introduction%20to%20machine%20learning&f=false/).
+
+### References
+1.	Kaelbling, L. P., Littman, M. L., & Moore, A. W. (1996). Reinforcement learning: A survey. Journal of artificial intelligence research, 4, 237-285.
+2.	Sutton, R. S., & Barto, A. G. (1998). Introduction to reinforcement learning (Vol. 135). Cambridge: MIT press.
+3.	Szepesvári, C. (2010). Algorithms for reinforcement learning. Synthesis lectures on artificial intelligence and machine learning, 4(1), 1-103.
+4.	Henderson, P., Islam, R., Bachman, P., Pineau, J., Precup, D., & Meger, D. (2017). Deep reinforcement learning that matters. arXiv preprint arXiv:1709.06560.
+5.	François-Lavet, V., Henderson, P., Islam, R., Bellemare, M. G., & Pineau, J. (2018). An introduction to deep reinforcement learning. arXiv preprint arXiv:1811.12560.
+6.	Arulkumaran, K., Deisenroth, M. P., Brundage, M., & Bharath, A. A. (2017). A brief survey of deep reinforcement learning. arXiv preprint arXiv:1708.05866.
+7.	Arulkumaran, K., Deisenroth, M. P., Brundage, M., & Bharath, A. A. (2017). Deep reinforcement learning: A brief survey. IEEE Signal Processing Magazine, 34(6), 26-38.
+8.	Watkins, C. J., & Dayan, P. (1992). Q-learning. Machine learning, 8(3-4), 279-292.
+9. Dang, Q. V. (2019, December). Reinforcement Learning in Stock Trading. In International Conference on Computer Science, Applied Mathematics and Applications (pp. 311-322). Springer, Cham.
+
+
+### Author Biography & Headshot image
```

Comment:
@cyberShaw please add this information into the author folder and create an author page 🚀

Reply:
Alright, would do. I'll add it 👌